src/endpoints/Account.ts

```diff
@@ -52,6 +52,14 @@ export default class Account extends Http {
     )) as IAccountConfirmationResult
   }
 
+  public async forgotPassword(params: IQuery): Promise<NoContentResult> {
```

Comment:
Let's be explicit about what parameters need to be provided to reset a password. Instead of asking for the generic `IQuery` type, let's create a new type `ResetPasswordParams`.

```
export interface ResetPasswordParams extends IQuery {
  user: {
    email: string
  }
}
```